rust/private/rustdoc.bzl

```diff
@@ -48,7 +48,8 @@ def rustdoc_compile_action(
         toolchain,
         crate_info,
         output = None,
-        rustdoc_flags = []):
+        rustdoc_flags = [],
+        use_param_file_for_rustc_args = True):
```

Comment:
Can you add a small unittest ensuring that when this flag is `True` prama files prefixed with `@` are found in argv of the action?

Reply:
Good catch, will do.

Reply:
I just tried doing that but it looks like `Action.argv` returns the actual arguments for the action before they are turned into a file, rather than the arguments passed to the executable...

Reply:
Wow, that's kinda crazy. I feel like this is a bug in Bazel... Is there a way we can at least test two different actions with and without this flag?

Reply:
I suppose not. That's the point of the "always" in this PR

Reply:
Yup this is an unfortunate and a known limitation of unittesting args. You can access the original `args` value from the unittest (https://docs.bazel.build/versions/main/skylark/lib/Action.html#args), but you cannot do anything with it other than setting it as args into some other action. So the only way we could test this is by writing all these args into a file and registering an action that reads the file and makes assertions. In other words, you cannot use unittest, you need an integration test.